leptonai/api/types.py

```diff
@@ -28,6 +28,20 @@ def _get_valid_shapes_printout() -> str:
     return ", ".join(VALID_SHAPES)
 
 
+# Spec to hold metadata
+class MetadataV2(BaseModel):
```

Comment:
do we have to call this V2 in SDK? The backend V2 will soon be renamed

Reply:
Yeah we don't have to - if we intend to remove the V2 in backend, and since V1 was never exposed, let me change the name.

Reply:
yeah, MetadataV2 is only a variable name in the backend. When it returns a json, it uses `metadata` as the key (for both V1 and V2).